src/ood-preview/script/watch.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+source_dirs="lib bin asset config lib/ data/"
+args=${*:-"bin/server.exe run"}
+cmd="dune exec ${args}"
+
+function sigint_handler() {
+  kill "$(jobs -pr)"
+  exit 1
+}
+
+trap sigint_handler SIGINT
+
+while true; do
+  make build
+  $cmd &
+  fswatch -r -1 $source_dirs
```

Comment:
I didn't have `fswatch` installed so we get stuck in a while-loop, could be add a quick `which fswatch` or something similar and print a message if it's not installed?

Reply:
Good idea :)